### PR TITLE
3b and 3d documentation updates

### LINKED ISSUE #159: Notebook 3b - Documentation Improvements
**Is your feature request related to a problem? Please describe.**

Current documentation issues to be addressed:

1. Where does the user get their Ionpath MIBItracker panel file?
2. What adjustments must be made to the file?
3. What are moly points, and problem do they solve?

**Describe the solution you'd like**
1. Give a brief description either in the notebook, or the README where the panel should be found
2. Describe what changes need to be made. Show an image or give a link to an example file for the initial panel, and the post adjustments panel.
3. Describe what moly points are used for, and how to know if you have them. In addition state that this is not needed for the newer instruments.

**Describe alternatives you've considered**
N/A

**Additional context**
View the Design Doc outlining the general improvements for all notebooks here: #157

### LINKED ISSUE #161: Notebook 3d - Documentation Improvements
**Is your feature request related to a problem? Please describe.**
Current documentation issues to be addressed:

1. Currently the MPH notebook is lacking details on what it does, and why it's important.

**Describe the solution you'd like**

1. Add the following details throughout the MPH Notebook: 
    1. _how_ is this happening (perhaps a brief description of whatever algorithm is being used)
    2.  _what_ happens when these cells are ran
    3.  _why_ (_significance_) do we need to do run this notebook
    1. Show figures of what to look out for (ideal), or describe it.
 
**Describe alternatives you've considered**
N/A

**Additional context**
View the Design Doc outlining the general improvements for all notebooks here: #157

### PR DESCRIPTION
**Pull request by camisowers**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #159 and closes #161.
Main changes:
* 3a : change intensity arg to have default ['Au', 'chan_39'] in watcher callbacks and remove from notebook
* 3b: remove intensity and replace args from notebook and make them defaults, edit print statements to let user know when extraction is complete
* 3d: more detailed mph descriptions and save mph plots to run_metrics (similar to qc notebook)
* deletes per_tof_histograms notebook which was replaced by the [compute pulse heights notebook](https://github.com/angelolab/toffy/blob/main/templates/compute_pulse_heights.ipynb)

### CODE REVIEW

File: templates/3b_extract_images_from_bin.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "# Extract Bin Files\n",
```
Comment (ngreenwald) [inline templates/3b_extract_images_from_bin.ipynb:10]: <p>This won't be necessary, just adds confusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ngreenwald) [inline templates/3b_extract_images_from_bin.ipynb:10]: <p>This applies for all notebooks, have code that needs to be modified in a separate cell, and indicate to the user that it needs to happen</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (ngreenwald) [inline templates/3b_extract_images_from_bin.ipynb:10]: <p>Remove from cell, documentation, and set as default for both this notebook and watcher extraction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: templates/3d_compute_median_pulse_height.ipynb [hunk 2]
```diff
@@ -7,7 +7,7 @@
    "source": [
```
Comment (ngreenwald) [inline templates/3d_compute_median_pulse_height.ipynb:7]: <p>Update this with Noah's new readme link about default values for MPH.</p><p> </p><p> </p><p>This section should tell the user that we use it to track the status of the detector, and when it gets otuside of the accepted range, know to perform a detector sweep to bring it back. </p><p> </p><p>This is a quantitative way to know when the instrument has been running long enough to need, or not need, a new sweep</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ngreenwald) [inline templates/3d_compute_median_pulse_height.ipynb:7]: <p>Move defaults to function defs, not at top of cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (ngreenwald) [inline templates/3d_compute_median_pulse_height.ipynb:7]: <p>Give name of directroy where plots are saved</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by camisowers** [comment]
watcher notebook documentation is being updated in the image stitching PR #152

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**